### PR TITLE
fixed some wrong translations

### PR DESCRIPTION
**Pull request by colttt**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #4508

I updated the translation:
dustbin -> trashbin
rehearsal -> repetition

### CODE REVIEW

**Comment by colttt** [comment]
uups ok, sorry, so you mean I need to create a new pull request?

File: inc/reservation.class.php [hunk 1]
```diff
@@ -760,7 +760,7 @@ function showForm($ID, $options = []) {
                       class='submit'>";
                if ($resa->fields["group"] > 0) {
                   echo "<br><input type='checkbox' name='_delete_group'>&nbsp;".
-                             __s('Delete all rehearsals');
+                             __s('Delete all repetition');
```
Comment (cedric-anne) [inline inc/reservation.class.php]: Missing a `s` for plural ?

**Comment by orthagh** [comment]
No need for another PR, just revert your change in locale folder and commit.

**Comment by orthagh** [comment]
@colttt, i'll do it for locales folder, don't bother :D

**Comment by orthagh** [comment]
ok you succeed to do it :)

**Comment by colttt** [comment]
yep; I'm amazed at myself ;-)

**Comment by orthagh** [comment]
Ok one last thing, and we will merge,.

I think `dustbined` need to be replaced by "put in trashbin", no ?

**Pull request merged**